x-pack/legacy/plugins/siem/server/lib/timeline/saved_object_mappings.ts

```diff
@@ -130,6 +130,9 @@ export const timelineSavedObjectMappings: {
       description: {
         type: 'text',
       },
+      eventType: {
+        type: 'keyword',
+      },
```

Comment:
We aren't GA yet...But if we have an old timeline data saved and it does not have an `eventType` which it wouldn't I would assume that we are going to either default it to `all` or we are going to begin writing our first migration code to deal with it?

If we do assume that the data type of undefined/null will be that way for older timelines we will have to make unit tests for it or some kind of guarantee that we never break it for potentially forever.

If we do not care about timelines pre-GA then disregard all of this, we don't care about pre-eventType timelines.

Reply:
The UI is managing that part

Reply:
meaning in there is no value it will consider that the value is raw since that was the only option at this time